src/cljs/lomakkeet/reagent/datepicker.cljs

```diff
@@ -74,26 +85,26 @@
                            ; Hide datepicker
                            (.hide @el)
                            nil)}
-             "×"]])])})))
+              "×"]])])})))
 
 (defn try-deref [x]
   (if (satisfies? IDeref x) @x x))
 
 (defn date* [form {:keys [ks datepicker-i18n min-date max-date date-time? clearable? disabled]}]
-  (let [this       (r/current-component)
+  (let [this (r/current-component)
         form-value (reaction (:value @(:data form)))
-        value      (reaction (get-in @form-value ks))]
+        value (reaction (get-in @form-value ks))]
     (fn [_ {:keys [ks datepicker-i18n min-date max-date date-time? disabled]}]
-      [date {:value @value
-             :on-blur #(impl/blur form ks)
-             :on-clear (fn [e]
-                         ;; Set date to nil
-                         (impl/cb form ks nil))
-             :on-select (fn [date]
-                          (impl/cb form ks date))
+      [date {:value           @value
```

Comment:
I prefer keeping map values (and let forms) unaligned, but not a huge issue. Aligning these causes lots of unnecessary changes when adding/renaming keys.